Vizzuality/gfw
# Chore/software updates

## Overview

This Pull Request upgrades both Nokogiri as well as Rails. There was a security vulnerability with the Nokogiri gem, and Rails released a new minor version, so we might as well update it! =)

## Testing

- get the branch
- run `bundle install`
- start the server
- everything should be the same as before.
- be proud of me. please. be.
